kernel/siw_cm.h

```diff
@@ -94,6 +99,9 @@ struct siw_cep {
 	struct siw_mpa_info	mpa;
 	int			ord;
 	int			ird;
+	int			revision;
+	int			p2ptype;
+	int			enhanced_rdma_connection;
```

Comment:
it is called 'enhanced rdma connection establishment' in rfc6581. so I think we do not have 'enhanced rdma' here, but some extended connection sequence with some extra info exchanged. call it mpav2info or so? Also, all three new parameters should better go into 'siw_mpa_info'...?